src/server/src/main/java/io/cassandrareaper/management/http/HttpCassandraManagementProxy.java

```diff
@@ -149,13 +172,22 @@ public int triggerRepair(
       List<RingRange> associatedTokens,
       int repairThreadCount)
       throws ReaperException {
-    return 1; //TODO: implement me
 
+    String jobId = "repair-1"; // TODO Make the actual implementation to call the mgmt-api to repair
```

Comment:
In the JMX implementation, the repairID is used to match the repair to the notifications, yes. Since I didn't want to rewrite that implementation, I simply mimicked the same behavior here. 

The jobId comes as the return payload when we submit the repair call, yes (just like all the other mgmt-api async calls do)